src/Genny.hpp

```diff
@@ -807,7 +815,8 @@ class Enum : public Type {
         Indent _{os};
 
         for (auto&& [name, value] : m_values) {
-            os << name << " = " << value << ",\n";
+            if (!m_type || 1ull << (m_type->size() * 8) > value)
```

Comment:
This is a good check. I would prefer if it was added to [SdkGenny](https://github.com/cursey/sdkgenny) though and not this projects copy of it.